### PR TITLE
macos 10.14 week 33 updates

### PR DESCRIPTION
**Pull request by jaskes**
Homebrew 2.1.9
    fastlane 2.128.1
    Miniconda 4.7.10
    azure-cli 2.0.70
    NVM node:
    v10.16.2
    v12.8.0
    xcversion 2.6.0
    android platform tools 29.0.2
    GNU parallel 20190722
    Visual Studio for Mac: 8.2.2.21
    Mono Version: 6.0.0
    Xamarin.iOS SDK Version: 12.14.0.114
    Xamarin.Android SDK Version: 9.4.1.0
    Xamarin.Mac SDK Version: 5.16.0.5

### CODE REVIEW

**Comment by v-mitim** [comment]
https://github.com/microsoft/azure-pipelines-image-generation/pull/1176/files#diff-fb7f82d5a779a32aabccaad5bb9ab35cR92
It's beta 5 already(Build version 11M382q)

**Comment by v-mitim** [comment]
ids for beta simulators also have been changed:
image

File: images/macos/macos-10.14-Readme.md [hunk 1]
```diff
@@ -160,7 +161,6 @@ Previously, Microsoft hosted Mac machines had JDKs pre-installed that were overl
 | iOS 9.1 (13B143)<br>iOS 9.2 (13C75)<br>iOS 9.3 (13E233) | 9.4.1, 10.0, 10.1, 10.2, 10.2.1 | iPhone 4s<br>iPhone 5<br>iPhone 5s<br>iPhone 6<br>iPhone 6 Plus<br>iPhone 6s<br>iPhone 6s Plus<br>iPad 2<br>iPad Retina<br>iPad Air<br>iPad Air 2<br>iPad Pro |
 | iOS 10.0 (14A345) | 9.4.1, 10.0, 10.1, 10.2, 10.2.1 | iPhone 5<br>iPhone 5s<br>iPhone 6<br>iPhone 6 Plus<br>iPhone 6s<br>iPhone 6s Plus<br>iPhone SE<br>iPad Air<br>iPad Air 2<br>iPad Pro (9.7 inch)<br>iPad Pro (12.9 inch) |
 | iOS 10.1 (14B72)<br>iOS 10.2 (14C89) | 9.4.1, 10.0, 10.1, 10.2, 10.2.1 | iPhone 5<br>iPhone 5s<br>iPhone 6<br>iPhone 6 Plus<br>iPhone 6s<br>iPhone 6s Plus<br>iPhone 7<br>iPhone 7 Plus<br>iPhone SE<br>iPad Air<br>iPad Air 2<br>iPad Pro (9.7 inch)<br>iPad Pro (12.9 inch) |
-| iOS 10.3 (14E8301) | 9.4.1, 10.0, 10.1, 10.2, 10.2.1 | iPhone 5<br>iPhone 5s<br>iPhone 6<br>iPhone 6 Plus<br>iPhone 6s<br>iPhone 6s Plus<br>iPhone 7<br>iPhone 7 Plus<br>iPhone SE<br>iPad Air<br>iPad Air 2<br>iPad Pro (9.7 inch)<br>iPad Pro (12.9 inch)<br>iPad (5th generation)<br>iPad Pro (12.9-inch) (2nd generation)<br>iPad Pro (10.5-inch) |
```
Comment (max-lobanov) [inline images/macos/macos-10.14-Readme.md]: Why Xcode 10.3 was removed?

Comment (max-lobanov) [inline images/macos/macos-10.14-Readme.md]: Why Xcode 10.3 was removed?

Comment (jaskes) [reply images/macos/macos-10.14-Readme.md]: isnt it IOS runtime?

Comment (max-lobanov) [reply images/macos/macos-10.14-Readme.md]: Sorry, just my mistake. Thought that Xcode 10.3 was removed.

**Pull request merged**